.github/workflows/production-build.yaml

```diff
@@ -1,11 +1,17 @@
 name: Unstract Docker Image Build and Push (Production)
 
 on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Docker image tag for the hotfix"
```

Comment:
- description should be "Docker image tag/branch for the build".  some time require manual build to test the image in various environment. 
- and this input is mandatory so no need a default value

Reply:
@jaseemjaskp, the mandatory part is clear.
but "Docker image tag/branch for the build" will always be a tag correct. tag for identifying the docker image.

Reply:
create on more input `branch` and add that as a `ref` variable value in manual triggering

Reply:
Addressed.